scripts/13-why-rust.md

```diff
@@ -0,0 +1,469 @@
+
+Sailors accepted that these were just the risks that you had to accept to operate on the high seas.
+
+---
+
+# Here Be Overheads
+
+notes:
+
+You know what else isn't boring? SOFTWARE.
+
+And I don't mean in a good way.
+
+In our field in 2022, everyone accepts that there are some things you just have to deal with in order to write software.
+
+---
+
+# Lies Programmers Believe About Software
+
+notes:
+
+some lies programmers believe about software, include, but are not limited to:
+
+- That software is built on unknowable layers of abstraction
+- The layers can only do what you want if you recite the correct inscrutable incantations
+- Scaling horizontally is the way to solve your problems
+- Bugs must be caught in production.
+- If you don't use yaml and javascript to solve your problems, you will have to use xml and java.
+- speed
+
+---
+
+## I Am Tired of Accepting Things I Cannot Change
+
+I am now changing things I cannot accept
+
+notes:
+
+I have been a web developer for 15 years, writing backends in python, ruby and node, and frontends in various javascript frameworks.
+
+I am tired of writing bullshit that I can't guarantee.
+
+and I'm not alone.
+
+---
+
+![[linus-easy-going.png]]
+
+(with apologies to Linus)
+
+notes:
+
+Linus Torvalds is not an easy man to please.
+
+As you probably know, Linus created and oversees the development of the Linux kernel.
+
+He is a detail-oriented man who cares about reliability over all else.
+
+And yet when Rust support was recently merged in to the Linux kernel, many veteran developers decried this step as bowing to a flashy, fashionable new language.
+
+The idea that Linus would approve of rust due to POPULARITY is laughable to those who know his reputation.
+
+---
+
+![[network diagram.png]]
+
+notes:
+
+And Linus is ABSOLUTELY RIGHT to be so conservative.
+
+Nearly every computer used show you this video, from your wifi router, to the network systems in your school, company and isp, to google's global video distribution network, is running this kernel.
+
+There are islands of mac devices. But Linux is the OCEAN.
+
+There are clouds of windows networks. But Linux is the SKY.
+
+%% with apologies to Audioslave %%
+
+---
+
+![[linus-chill.png]]
+notes:
+
+We are not talking about your pet projects, or your companies production projects.
+
+We're not even talking about real-time banking software and hospital systems.
+
+We are talking about Linux.
+
+Rust is not the first language to be tried as an alternative to C, to write the linux kernel.
+
+C++ was proposed 25 years ago, but failed to meet Linus's extremely high standards
+
+C++ is not good enough for the Linux kernel, but Rust is.
+
+---
+
+And Rust, despite being a better language to write the Linux kernel then C++ is also a fantastic web development language.
+
+---
+
+## 🚀 Rocket.rs
+
+![[rocket-rs.png]]
+
+notes:
+
+Rocket is one of a handful of stable feature-rich backend web development frameworks rocket has
```

Comment:
"rocket has" at the end doesn't make sense? (maybe it's just leftover from modifications)

I suggest adding "for Rust" instead
```suggestion
Rocket is one of a handful of stable feature-rich backend web development frameworks for Rust.
```